compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -58,6 +58,12 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer { thisPhase
   /** the following two members override abstract members in Transform */
   override def phaseName: String = PostTyper.name
 
+  override def checkPostCondition(tree: tpd.Tree)(implicit ctx: Context): Unit = tree match {
+    case tree: ValOrDefDef =>
+      assert(!tree.symbol.signature.isUnderDefined)
```

Comment:
Do we need to match `DefDef` here instead of `ValOrDefDef`?

Reply:
I used this to all cover signatures of getters.